Themis/Views/Assessment/CorrectionSidebar/EditFeedbackView.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  EditFeedbackView.swift
+//  Themis
+//
+//  Created by Katjana Kosic on 28.11.22.
+//
+
+import Foundation
+import SwiftUI
+
+struct EditFeedbackView: View {
+    @ObservedObject var feedbackModel: FeedbackViewModel
+    @State var feedbackText = ""
+    @State var score = 0.0
+    @Binding var showEditFeedback: Bool
+    var feedbackID: Feedback.ID? // if nil add sheet else edit
+    let maxScore = Double(10)
+    let artemisColor = Color(#colorLiteral(red: 0.20944947, green: 0.2372354269, blue: 0.2806544006, alpha: 1))
+
+    var pickerRange: [Double] {
+        Array(stride(from: -1 * maxScore, to: maxScore +  0.5, by: 0.5))
+            .sorted { $0 > $1 }
+    }
+    var title: String {
+        feedbackID == nil ? "Add Feedback" : "Edit Feedback"
+    }
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack {
+                Text(title)
+                    .font(.largeTitle)
+                Spacer()
+                Button {
+                    withAnimation {
+                        save()
+                        showEditFeedback = false
+                    }
+                } label: {
+                    Text("Save")
```

Comment:
Indicate that the Button is disabled by setting a dynamic foregroundColor
```suggestion
                    Text("Save")
                    .foregroundColor(feedbackText.isEmpty ? .gray : .blue)
```